lib/supavisor_web/ws_proxy.ex

```diff
@@ -61,6 +61,6 @@ defmodule SupavisorWeb.WsProxy do
   @spec connect_local() :: {:ok, port()} | {:error, term()}
   defp connect_local() do
     proxy_port = Application.fetch_env!(:supavisor, :proxy_port_transaction)
-    :gen_tcp.connect('localhost', proxy_port, [:binary, packet: :raw, active: true])
+    :gen_tcp.connect(~c"localhost", proxy_port, [:binary, packet: :raw, active: true])
```

Comment:
Is there any benefit to using sigils here?

Reply:
ahh, it will be deprecated, right

Reply:
Yes, deprecation warnings.